src/GraphQL.SystemTextJson/InputsConverter.cs

```diff
@@ -1,22 +1,105 @@
 using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 
 namespace GraphQL.SystemTextJson
 {
     /// <summary>
     /// A custom JsonConverter for reading an <see cref="Inputs"/> object.
-    /// Doesn't support write.
     /// </summary>
     public class InputsConverter : JsonConverter<Inputs>
     {
         /// <inheritdoc/>
         public override void Write(Utf8JsonWriter writer, Inputs value, JsonSerializerOptions options)
-            => throw new NotImplementedException(
-                "This converter currently is only intended to be used to read a JSON object into a strongly-typed representation.");
+            => JsonSerializer.Serialize<IReadOnlyDictionary<string, object>>(writer, value, options);
```

Comment:
For a single line of code, there's no reason not to support this functionality, even if it is rarely (if ever) used.